config/initializers/filter_parameter_logging.rb

```diff
@@ -1,6 +1,8 @@
 # Be sure to restart your server when you modify this file.
 
-# Configure sensitive parameters which will be filtered from the log file.
+# Configure parameters to be partially matched (e.g. passw matches password) and filtered from the log file.
+# Use this to limit dissemination of sensitive information.
+# See the ActiveSupport::ParameterFilter documentation for supported notations and behaviors.
 Rails.application.config.filter_parameters += [
-  :password, :passw, :secret, :token, :_key, :crypt, :salt, :certificate, :otp, :ssn, :query
+  :password, :passw, :secret, :token, :_key, :crypt, :salt, :certificate, :otp, :ssn
```

Comment:
We intentionally include `query` here to keep from logging potentially sensitive query terms.  This needs to be added back.